code/game/area/areas/ruins/wasteplanet.dm

```diff
@@ -1 +1,43 @@
 //area defs go here :)
+
+//Abandoned Mechbay Ruin
+
```

Comment:
10 areas is a little excessive.

Reply:
somewhere along the line my attempts at cutting down got reverted. Whoops.